fendermint/actors/chainmetadata/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+mod actor;
+mod shared;
```

Comment:
I don't fully understand how this one is actually _shared_ with the normal Rust code, it all looks like one crate, whereas for example the `evm` actor case [shared](https://github.com/filecoin-project/builtin-actors/tree/master/actors/evm/shared) has its own `Cargo.toml` file.

Reply:
yeah, maybe it was split up in the `evm` actor to reduce code compilation?

Reply:
Yes, if it's split it doesn't (shouldn't?) need reference to certain things like the actor runtime, so we can import just the code to deal with the DTOs and the state for example, but not all the other stuff that the actual actor implementation can require.